admin-ui/app/template/inner-layout/inner-header/InnerHeaderComponent.tsx

```diff
@@ -73,19 +71,12 @@ const InnerHeaderComponent = ({
   const innerSectionData = dragList.map((ele: any) => ele.properties);
 
   const saveJsonData = () => {
-    // console.log("JSON ", [
-    //   { projectName: project[0].name },
-    //   { component: dragList },
-    //   { property: innerSectionData },
-    // ]);
-
     var json = JSON.stringify([
```

Comment:
no need to have stringfy, you have to pass type to service method and that will take care for this